engine/src/dsklnx.cpp

```diff
@@ -110,7 +110,11 @@ static void parseSerialControlStr(char *setting, struct termios *theTermios)
         if (MCU_strncasecmp(type, "baud", strlen(type)) == 0)
         {
             long baudrate = strtol(value, NULL, 10);
-            if (baudrate == 57600)
+            if (baudrate == 230400)
```

Comment:
It would be neater to structure this as:
```
switch(baudrate)
{
case NNNNN: baud = BNNNNN; break;
default:
  baud = B9600;
  break;
}
```
Note: B9600 as the default is entirely arbitrary - although it is suggested that is a 'sensible' default after a quick google.